windows_kext/release/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "release"
+version = "0.1.0"
```

Comment:
Should this version match version.txt?

Reply:
This is used only by the rust build system.
It's only necessary if you want to use it with the package manager.
The libraries in the repository are refer by relative path, so the version here does not matter.